app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+from app.players.player import Player
+from app.players.elves.elf_ranger import Elf
+from app.players.dwarves.dwarf_blacksmith import Dwarf
+
+
+def calculate_team_total_rating(players: list[Player]) -> int:
+    return sum(player.get_rating() for player in players)
+
+
+def elves_concert(elves: list[Elf]) -> None:
+    [elf.play_elf_song() for elf in elves]
```

Comment:
Try to avoid using list compl when you do not use list

Reply:
Hello. Do you mean to change it to a simple for loop?

    for elf in elves:
        elf.play_elf_song()